extensions/wikia/AdEngine/js/video/videoSettings.js

```diff
@@ -28,6 +28,8 @@ define('ext.wikia.adEngine.video.videoSettings', [
 		function calculateMoatTrackingFlag(params) {
 			var sampling = params.moatTracking || 0;
 
+			debugger
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this debugger statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ADebuggerStatement)